src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration.mdx

```diff
@@ -1,21 +1,25 @@
 ---
-title: Amazon API Gateway monitoring integration
+title: Amazon API Gateway V1 monitoring integration
 tags:
   - Integrations
   - Amazon integrations
   - AWS integrations list
 metaDescription: "New Relic's AWS API Gateway monitoring integration: what data it reports, and how to enable it."
 redirects:
-  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-api-gateway-monitoring-integration
+  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-api-gateway-v1-monitoring-integration
```

Comment:
@rahul188 I don't think we should change this because there has never been a document at this location (with the v1). If we simply leave this as it was, any old links will still resolve to this v1 page, which is what we want, right?

Reply:
@rhetoric101, I made the suggested change.